js/lib/monster.js

```diff
@@ -25,7 +23,7 @@ var Sprite = require('./sprite');
         } else {
           spriteVersion += 0.1
         }
-        if (xDiff >= 0) {
+        if (that.movingToward[0] > that.mapPosition[0]) {
           return 'sEast' + Math.ceil(spriteVersion)
         } else if (xDiff < 0) {
```

Comment:
`xDiff` isn't set anymore, this caused the game to crash while testing:

```
Uncaught ReferenceError: xDiff is not defined
    at spritePartToUse (monster.js:28)
    at Sprite.Monster.that.draw (monster.js:33)
    at SpriteArray.<anonymous> (game.js:111)
    at S (sugar-1.3.7.min.js:27)
    at SpriteArray.each (sugar-1.3.7.min.js:37)
    at Game.that.draw (game.js:110)
    at Game.step (game.js:172)
```

Reply:
Of course, that should be an `else` now, thanks!

I might have mucked that up when resorting commits for a rebase.